packages/trader/src/Modules/Contract/Components/Digits/digits.tsx

```diff
@@ -77,24 +122,24 @@ const DigitsWrapper = ({
     );
 };
 
-const Digits = React.memo(props => {
-    const [status, setStatus] = React.useState();
-    const [current_tick, setCurrentTick] = React.useState();
-    const [spot, setSpot] = React.useState();
-    const [is_selected_winning, setIsSelectedWinning] = React.useState();
-    const [is_latest, setIsLatest] = React.useState();
-    const [is_won, setIsWon] = React.useState();
-    const [is_lost, setIsLost] = React.useState();
+const Digits = React.memo((props: TDigits) => {
+    const [status, setStatus] = React.useState<string | null>();
+    const [current_tick, setCurrentTick] = React.useState<number | null>();
+    const [spot, setSpot] = React.useState<string>();
+    const [is_selected_winning, setIsSelectedWinning] = React.useState<boolean>();
+    const [is_latest, setIsLatest] = React.useState<boolean>();
+    const [is_won, setIsWon] = React.useState<boolean>();
+    const [is_lost, setIsLost] = React.useState<boolean>();
```

Comment:
```suggestion
    const [is_lost, setIsLost] = React.useState<boolean | undefined>();
```
@henry-deriv initial value is undefined, is TS not complaining about it?

Reply:
not complaining since it can detect the type i guess

`const setIsLost: React.Dispatch<React.SetStateAction<boolean | undefined>>`